e2e_tests/test_admin.py

```diff
@@ -21,8 +26,41 @@ def add_task(base_url: str, page: Page, title, parent=None):
     page.get_by_role("button", name="Sichern", exact=True).first.click()
 
 
-def drag_task(page, task_to_drag: str, target_task: str):
+def drag_task_on_task_cap_board(
+    page: Page, task_to_drag_title: str, target_task_title: str
+):
+    task_to_drag = page.get_by_text(task_to_drag_title)
+    target_task = page.locator(".cap_board__task-item", has_text=target_task_title)
+    target_task.scroll_into_view_if_needed()
+
+    to_drag_x, to_drag_y = get_position_in_the_middle(task_to_drag)
+    target_x, target_y = get_position_in_the_lower_third(target_task)
+
+    page.mouse.move(to_drag_x, to_drag_y)
+    page.mouse.down()
+    page.mouse.move(target_x, target_y)
+    page.mouse.up()
+
+
+def drag_task_above_task_cap_board(
+    page: Page, task_to_drag_title: str, target_task_title: str
+):
```

Comment:
I think here we should make more clear what's the difference between this two methods. What do you think about `_below_` instead of `_on_` (if I unterstood it correctly)? Additionally it would be more readable when `_cap_board` is not a postfix: `drag_cap_board_task_below_task` and `drag_cap_board_task_above_task`. What do you say?

Reply:
The question is whether you want to name it according to what you do i.e. drag a task on another task or what happens i.e. the task is added in the hierarchy below the task it was dragged on to. 
I decided for the first as the e2e tests describe what a user would do. 
I also do not really think of the cap_board as a postfix but as part of the sentence: "drag a task on a task on the cap board". (But maybe I should but an "on_the_" into the name then) When you put it as in your example it reads as a description of the task: "drag a cap board task on as task" and then I would wonder what are "cap board tasks" vs "tasks". 
Does it make more sense now?

Reply:
Yes perfect